packages/core/src/App/Containers/AccountSwitcher/account-switcher-account-list.jsx

```diff
@@ -15,18 +15,22 @@ const AccountList = ({
     is_disabled,
     is_virtual,
     loginid,
-    market_type,
-    onClickAccount,
+    account_type,
+    redirectAccount,
     onClickResetVirtualBalance,
     selected_loginid,
     server,
     is_dark_mode_on,
     sub_account_type,
     platform,
 }) => {
+    const switchAccountType = async acc_type => {
+        await redirectAccount();
+        window.location.hash += `-${acc_type}`;
```

Comment:
1. It should be `market_type` since it represents `financial|gaming`
2. Can we pass `market_type` to `redirectAccount` fn and change the hash in a single go in `redirectToDXTrade` and `redirectToMt5`?

```suggestion
    const switchAccountType = async market_type => {
        await redirectAccount(market_type);
```

Reply:
Updated